src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenNullCheckedParameterTests.cs

```diff
@@ -772,7 +722,73 @@ static void Invoke(Action a)
 }";
             CompileAndVerify(source, expectedOutput: @"
 ok
-System.Exception");
+System.ArgumentNullException");
+        }
+
+        [Fact]
+        public void NotNullCheckedLambdaParameter()
```

Comment:
>NotNullCheckedLambdaParameter [](start = 20, length = 29)

The same test is here several times, and the test does not include null checking.